src/widgets/survey/model/mappers.ts

```diff
@@ -1,46 +1,69 @@
-import { Answers, PipelineItem } from '@app/features/pass-survey';
+import {
+  Answers,
+  PipelineItem,
+  PipelineItemAnswer,
+} from '@app/features/pass-survey';
 import { AnswerDto } from '@app/shared/api';
 
-export default function isObject<TObj>(obj: TObj) {
-  var type = typeof obj;
-  return type === 'function' || (type === 'object' && !!obj);
-}
+type Answer = PipelineItemAnswer['value'];
 
 export function mapAnswersToDto(
   pipeline: PipelineItem[],
   answers: Answers,
-  flowId: string | null,
 ): Array<AnswerDto> {
   const filteredAnswers = Object.entries(answers).filter(([_, answer]) => {
```

Comment:
Just a comment.
When conditional logic is merged we should all add another filter that filters items that cannot pass the conditional logic.